%________________________________________________________________________________________
% @brief    LaTeX2e Resume for Camille Scott
% @author   Camille Scott
% @date     
% @info     Based on Latex Resume Template by Chris Paciorek 
%           http://www.biostat.harvard.edu/~paciorek/


%________________________________________________________________________________________
\documentclass[resmargin,12pt]{camille_resume}

\usepackage{hyperref}
\usepackage{color}
\hypersetup{urlcolor=blue}

\begin{document}
\name{\Large Camille Scott}
\address{{\bf Permanent Address}\\
By Request\\
Davis, CA}
\address{
\hfill {\bf Contact}\\
\hfill cswel@ucdavis.edu\\
\hfill camille.scott.w@gmail.com
}
\pagenumbering{gobble}
\pagestyle{plain}

    
\begin{resume}


    %____________________________________________________________________________________

\section{\small \it{(as of May 2023)}}

% Education
\section{\mysidestyle Education}

{\bf University of California Davis}, Davis, CA \hfill 2015-2022\\
{\em PhD., Computer Science, Lab for Data Intensive Biology}\\
{\em Dissertation: \href{https://github.com/camillescott/dissertation}{Streaming Methods for
    Assembly Graph Analysis}}

\vspace{1mm}

{\bf Michigan State University}, East Lansing, MI \hfill 2012-2015\\
{\em GED Lab, PhD. student, Computer Science}

\vspace{1mm}

{\bf Alma College}, Alma, MI \hfill 2008-2012\\
{\em B.S., Computer Science}

\vspace{2mm}

\section{\mysidestyle Work}

\begin{list1}

\item[] Systems Administrator at {\bf UC Davis HPCCF} \hfill 7/2022-present\\\\
\begin{workdesc}
Provide scientific computing support to the College of Biological Sciences for the High Performance Computing Core Facility.
Co-lead design and deployment of the Franklin cluster consisting of 11 compute and GPU nodes with 2PB of attached network storage:
designed and maintain reproducible software stack for genomics and cryo-EM workflows using Spack and Conda; design and maintenance of
core software and libraries for scheduling and infrastructure; co-designed and implemented complete refactoring of Puppet configuration
codebase for automated configuration management; communicate and implement stakeholder and user needs; collaborate with researchers on deployment,
optimization, and use of their scientific software in the HPC environment; deploy and maintain user documentation and tutorials.
\end{workdesc}

\item[] Genomic Data Specialist at {\bf APHL} \hfill 5/2021-8/2022\\\\
\begin{workdesc}
Processed SARS-CoV-2 genomes through contract work mediated by the Association of Public Health Labs in the COVID-19 pandemic effort. 
Quality controlled genome assemblies, managed genome metadata, wrote processing workflows, and submitted genomes to GISAID for public archiving.
\end{workdesc}

\item[] Teaching Assistant at {\bf UC Davis}\hfill Spring 2021\\\\
\begin{workdesc}
ECS132: Probabilistic and Statistical Modeling in Computer Science. Graded assignments and exams, held office hours, gave one-on-one student help, 
and developed course materials.
\end{workdesc}

\item[] Teaching Assistant at {\bf UC Davis} \hfill Winter 2021\\\\
\begin{workdesc}
GGG201B: Genomics. Provided student help and graded take-home assignments.
\end{workdesc}

\item[] Junior Specialist at {\bf UC Davis} \hfill 6/2015-10/2015\\\\
\begin{workdesc}
Scientific software development for the khmer project; worked on core functionality (high performance 
C++ with CPython bindings, Python CLI), automated testing, continuous integration, packaging and distribution.
\end{workdesc}

\item[] Workshop Teaching Assistant at {\bf Marine Biological Laboratory}\hfill 8/2013\\\\
\begin{workdesc}
TA for the two-week intensive Strategies and Techniques for Analyzing Microbial Population Structures.
Assisted students with hands-on bioinformatics training, developed materials, and taught portions.
\end{workdesc}

\item[] Lead Instructor at {\bf Michigan State University} \hfill Fall 2012\\\\
\begin{workdesc}
CSE101: Primary instructional interface for three sections of 30 undergraduates each; 
delivered lecture, graded assignments and exams, held office hours and study sessions.
\end{workdesc}

\item[] Teaching Assistant (TA) at {\bf Alma College}\hfill 2011-2012\\\\
\begin{workdesc}
Intro to Computer Science. Assisted students during programming laboratory sessions.
\end{workdesc}

\end{list1}


%____________________________________________________________________________________
% Publications


\section{\mysidestyle Manuscripts}

Scott, C., \& Brown, C. T. (2022). {\em Streaming Construction of the Compact de Bruijn Graph}. In prep.

Reiter, T., Brooks, P. T., Irber, L., Joslin, S. E., Reid, C. M., Scott, C., ... \& Pierce, N. T. (2020). {\em Streamlining Data-Intensive Biology With Workflow Systems.}. BioRxiv.

Neches, R. Y., \& Scott, C. (2018). {\em SuchTree: Fast, thread-safe computations with phylogenetic trees.} Journal of Open Source Software, 3(26), 678.

Scott, C. (2017). {\em shmlast: An improved implementation of Conditional Reciprocal Best Hits with LAST and Python.} The Journal of Open Source Software, 2017.

Ren, J., Chung-Davidson, Y. W., Yeh, C. Y., Scott, C., Brown, T., \& Li, W. (2015). {\em Genome-wide analysis of the ATP-binding cassette (ABC) transporter gene family in sea lamprey and Japanese lamprey.} BMC genomics, 16(1), 436.

Crusoe, M. R. et. al. (2015). {\em The khmer software package: enabling efficient nucleotide sequence analysis.} F1000Research, 4.

Aleksic, J., Alexa, A., Attwood, T. K., Hong, N. C., Dahl√∂, M., Davey, R., ..., \& Lahti, L. (2014)
{\em The open science peer review oath.} F1000Research, 3.

\pagebreak

%____________________________________________________________________________________
% Honours and Awards
\section{\mysidestyle Honors and\\Awards} 

USDA / MSU /  National Needs Fellow \hfill 2013-2015\\
Binghamton University / Research Experience for Undergrads \hfill 2011\\
Alma College / Distinguished Scholar Award \hfill 2008-2012\\

%____________________________________________________________________________________
\section{\mysidestyle Professional\\Activities}

\begin{list1}
\item[] Lead Instructor for workshop at {\bf UC Davis},\\
{\em \href{https://angus.readthedocs.io/en/2017/}{ANGUS: Next-Gen Sequence Analysis Workshop}} \hfill Summer 2017\\

\item[] Participant in Workshop at {\bf University of California Davis},\\
{\em Software Carpentry ``Train the Trainers" } \hfill 01/2015\\

\item[] Participant in Workshop at {\bf The Genome Analysis Center (TGAC)},\\
{\em AllBio Open Science \& Reproducibility Best Practices Workshop} \hfill 09/2014\\

\item[] TA for SROP Workshop at {\bf Michigan State University}, \\
{\em Statistics Bootcamp with the Summer Research Opportunities Program} \hfill 05/2014\\

\item[] Presenter at {\bf Michigan State University},\\
{\em Summer Research Opportunities (SROP) Workshop} \hfill 5/2013\\

\item[] Workshop TA at {\bf California Institute of Technology},\\
\em{Workshop on Microbial Bioinformatics} \hfill 10/2013\\

\item[] Co-founder, {\bf Alma College Association for Computing Machinery} \hfill 2012\\


\end{list1}


   %____________________________________________________________________________________
    \section{\mysidestyle Software}

\begin{list1}

\item[] Author and maintainer of {\bf cheeto},\\
{\em Python library and CLI tools for HPC management}\\
\hfill \href{https://github.com/ucdavis/cheeto}{github.com/ucdavis/cheeto}\\

\item[] Author and maintainer of {\bf goetia},\\
{\em C++ and Python library and command line tools for streaming sequence analysis}\\
\hfill \href{https://github.com/camillescott/goetia}{github.com/camillescott/goetia}\\

\item[] Author and maintainer of {\bf debruijnal-enhance-o-tron},\\
{\em pytest fixtures for random de Bruijn graph generation}\\
\hfill \href{https://github.com/camillescott/debruijnal-enhance-o-tron}{github.com/camillescott/debruijnal-enhance-o-tron}\\

\item[] Author and maintainer of {\bf dammit},\\
{\em a tool for easy de novo transcriptome annotation}\\
\hfill \href{https://github.com/camillescott/dammit}{github.com/camillescott/dammit}\\

\item[] Author and maintainer of {\bf shmlast},\\
{\em an implementation of conditional reciprocal best hits with Python and LAST}\\
\hfill \href{https://github.com/camillescott/shmlast}{github.com/camillescott/shmnlast}\\

\item[] Contributor to {\bf khmer Protocols Project},\\
{\em Protocols for cloud-based de novo RNA-seq and metagenomic analyses}\\
\hfill \href{https://khmer-protocols.readthedocs.org/}{khmer-protocols.readthedocs.org}\\

\item[] Contributor to open source {\bf khmer software package},\\ 
{\em k-mer counting, graph traversal, and sequence processing}\\
\hfill \href{https://github.com/ged-lab/khmer/}{github.com/dib-lab/khmer}\\

\item[] Member of and contributor to {\bf bioconda channel},\\
{\em conda channel for bioinformatics packages for Linux and MacOS}\\
\hfill \href{https://github.com/bioconda/bioconda-recipes}{github.com/bioconda/bioconda-recipes}\\



\end{list1}
%________________________________________________________________________________________

% \newpage

\section{\mysidestyle Blogs and social media}

GitHub: \href{https://github.com/camillescott}{github.com/camillescott}, \href{https://github.com/camillescottatwork}{github.com/camillescottatwork}

Blog: \href{http://camillescott.org}{camillescott.org}  

\section{\mysidestyle References}

{\em (Contact details available upon request)}

\end{resume}

\end{document}

%________________________________________________________________________________________
% EOF


