*(as of March 2018)*
====================

Education
=========

| **Alma College**, Alma, MI 2008-2012
| *B.S., Computer Science*

| **Michigan State University**, East Lansing, MI 2012-2015
| *DIB Lab, PhD. student, Computer Science*

| **University of California Davis**, Davis, CA 2015-present
| *DIB Lab, PhD. student, Computer Science*

Honours and
Awards
===========

| National Merit Finalist 2008
| Alma College / Distinguished Scholar Award 2008-2012
| Binghamton University / Research Experience for Undergrads summer 2011
| USDA / MSU/ National Needs Fellow 2013-2015

Manuscripts
===========

Scott, C. (2017). *shmlast: An improved implementation of Conditional
Reciprocal Best Hits with LAST and Python*. The Journal of Open Source
Software, 2017.

Ren, J., Chung-Davidson, Y. W., Yeh, C. Y., Scott, C., Brown, T., & Li,
W. (2015). *Genome-wide analysis of the ATP-binding cassette (ABC)
transporter gene family in sea lamprey and Japanese lamprey.* BMC
genomics, 16(1), 436.

Crusoe, M. R. et. al. (2015). *The khmer software package: enabling
efficient nucleotide sequence analysis.* F1000Research, 4.

Aleksic, J., Alexa, A., Attwood, T. K., Hong, N. C., Dahlö, M., Davey,
R., ..., & Lahti, L. (2014) *The open science peer review oath.*
F1000Research, 3.

Blogs and social media
======================

Professional blog at:
`camillescott.github.io/blog <http://camillescott.github.io/blog/>`__

Twitter: `@camille\_codon <http://twitter.com/camille_codon>`__

Github: `github.com/camillescott <https://github.com/camillescott>`__

Professional
Activities
============

| Teaching Assistant (TA) at **Alma College**,
| *Intro to Computer Science* 2011-2012

| Co-founder, **Alma College Association for Computing Machinery** 2012

| Lead Instructor/TA at **Michigan State University**,
| *CSE 101* Fall 2012

| Presenter at **Michigan State University**,
| *Summer Research Opportunities (SROP) Workshop* 5/2013

| TA for workshop at **Marine Biological Laboratory**,
| *Strategies and Techniques for Analyzing Microbial Population
  Structures* 8/2013

| TA for workshop at **California Institute of Technology**,
| *Workshop on Microbial Bioinformatics* 10/2013

| TA for SROP Workshop at **Michigan State University**,
| *Statistics Bootcamp* 05/2014

| Participant in Workshop at **The Genome Analysis Center (TGAC)**,
| *AllBio Open Science & Reproducibility Best Practices Workshop*
  09/2014

| Participant in Workshop at **University of California Davis**,
| *Software Carpentry “Train the Trainers"* 01/2015

| Contributor to **khmer Protocols Project**,
| *Protocols for cloud-based de novo RNA-seq and metagenomic analyses*
| `khmer-protocols.readthedocs.org <https://khmer-protocols.readthedocs.org/>`__

| Contributor to open source **khmer software package**,
| *k-mer counting, graph traversal, and sequence processing*
| `github.com/dib-lab/khmer <https://github.com/ged-lab/khmer/>`__

| Author and maintainer of the **dammit** annotator,
| *a tool for easy de novo transcriptome annotation*
| `github.com/camillescott/dammit <https://github.com/camillescott/dammit>`__

| Author and maintainer of **shmlast**,
| *an implementation of conditional reciprocal best hits with Python and
  LAST*
| `github.com/camillescott/shmnlast <https://github.com/camillescott/shmlast>`__

References
==========

*(Contact details available upon request)*
